common-money.lic

```diff
@@ -64,11 +101,65 @@ module DRCM
   def check_wealth(currency)
     DRC.bput("wealth #{currency}", /\(\d+ copper #{currency}\)/i, /No #{currency}/i).scan(/\d+/).first.to_i
   end
-
   def wealth(hometown)
```

Comment:
Oops, I suggested an entire line be deleted when I meant to suggest just remove the whitespace. This suggestion adds the blank line back (no whitespace) between the `check_wealth` and `wealth` methods.
```suggestion
  end

  def wealth(hometown)
```

Reply:
> Applying suggestions on deleted lines is currently not supported.

I'll fix and commit.

Reply:
Added back blank line. Also fixed the regex in .scan.